crates/precompile/benches/eip2537.rs

```diff
@@ -0,0 +1,332 @@
+use ark_bls12_381::{Fq, Fr, G1Affine, G2Affine};
+use ark_ec::AffineRepr;
+use ark_serialize::CanonicalSerialize;
+use arkworks_general::{encode_base_field, encode_field_32_bytes, random_field, random_points};
+use criterion::{measurement::Measurement, BenchmarkGroup};
+use primitives::Bytes;
+use rand::{rngs::StdRng, SeedableRng};
+use revm_precompile::bls12_381_const::{
+    FP_LENGTH, FP_PAD_BY, PADDED_FP_LENGTH, PADDED_G1_LENGTH, PADDED_G2_LENGTH,
+};
+
+const RNG_SEED: u64 = 42;
+const MAX_MSM_SIZE: usize = 16;
+const MAX_PAIRING_PAIRS: usize = 10;
```

Comment:
These are arbitrary, MAX_MSM_SIZE might be better at 256 for the realistic maximum that will happen in the EVM?

For Pairing, 1/2 is most common, but 8/16 sound reasonable as a max

Reply:
changed to 256 and 16, and also added cases for MSM and pairing of size 2